Gemfile.lock

```diff
@@ -284,6 +284,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86_64-darwin-19
```

Comment:
This change happened because Bundler 2.2.0 [enables the `specific_platform` option by default](https://github.com/rubygems/rubygems/pull/4015) to "properly support gems providing native platform versions with different dependencies than their pure ruby counterparts."

As far as I understand, this should make it easier/faster to run `bundle install` by not requiring that gems like `nokogiri` be compiled from source.

I'd expect if someone with an older Mac or an Apple Silicon one were to run `bundle install`, they'd update the file with their platform, too. This makes me wary that we might get a confused message every now and again by a developer that set up the project and resulted in changes they didn't expect, _exactly what this PR is trying to solve_.

We can prevent this at the `.bundle/config` level by adding `BUNDLE_FORCE_RUBY_PLATFORM: "true"`.

But then I wonder if we'd be leaving precious seconds or minutes on the table when running `bundle install` just to avoid an occasional change to the file 🤔 

Then again, we don't often run `bundle install`, so maybe the time penalty of always building from source is an acceptable compromise for easier maintenance? 🤔 🤔 CI caches the gems so we shouldn't worry about performances there, either.

_What do you think?_

![](https://media.giphy.com/media/14mgxYFJHXGmoo/source.gif)

Reply:
I'll proceed to update the other repos once we'll make a decision on this 👍

Reply:
This seems like a win to avoid recompiling a lot on developer machines.

I don't see an issue with occasionally dirtying this file when running on ARM (I don't see anyone bundling this with Windows or Linux anytime soon...)

This seems ok to me as-is?

Reply:
Does this support multiple binaries platforms in parallel? I mean, what will happen when Apple Silicon macs will start to be used as development machines? People will always have a dirty repository or will it be possible to add the new platform without deleting x86_64?

Reply:
I just `bundle install`'d on WordPressAuthenticator with `bundler 2.2.1` (since that's the version its `Gemfile.lock` was already locking on) and [it added additional darwin platform to the ones that was already there in this repo](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/549/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR88-L88).

So at least it means that Bundler seems to support multiple binaries platforms in parallel, yes.

Reply:
Oh, in this case it looks good 👍